api/app/Models/CollectionStats.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class CollectionStats extends Model{
+
+	protected $table = 'omcollectionstats';
+	protected $primaryKey = 'collid';
+    const CREATED_AT = 'initialtimestamp';
```

Comment:
This seemed like a cool solution to the create and update timestamps being present in the table but not named according to Laravel convention.

Reply:
Fixed in 7f74c5564.